sw/device/tests/aes_test.c

```diff
@@ -16,10 +16,21 @@ static const uint8_t plain_text_1[16] = {0x00, 0x11, 0x22, 0x33, 0x44, 0x55,
                                          0x66, 0x77, 0x88, 0x99, 0xaa, 0xbb,
                                          0xcc, 0xdd, 0xee, 0xff};
 
-static const uint8_t key_32_1[32] = {
-    0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x0a,
-    0x0b, 0x0c, 0x0d, 0x0e, 0x0f, 0x10, 0x11, 0x12, 0x13, 0x14, 0x15,
-    0x16, 0x17, 0x18, 0x19, 0x1a, 0x1b, 0x1c, 0x1d, 0x1e, 0x1f};
+// The mask share
+static const uint8_t key_32_1_share1[32] = {
+    0x0f, 0x1f, 0x2f, 0x3F, 0x4f, 0x5f, 0x6f, 0x7f, 0x8f, 0x9f, 0xaf,
+    0xbf, 0xcf, 0xdf, 0xef, 0xff, 0x0a, 0x1a, 0x2a, 0x3a, 0x4a, 0x5a,
+    0x6a, 0x7a, 0x8a, 0x9a, 0xaa, 0xba, 0xca, 0xda, 0xea, 0xfa};
+
+// The masked key share (key XOR mask)
+static const uint8_t key_32_1_share0[32] = {
+    0x00 ^ 0x0f, 0x01 ^ 0x1f, 0x02 ^ 0x2f, 0x03 ^ 0x3f, 0x04 ^ 0x4f,
+    0x05 ^ 0x5f, 0x06 ^ 0x6f, 0x07 ^ 0x7f, 0x08 ^ 0x8f, 0x09 ^ 0x9f,
+    0x0a ^ 0xaf, 0x0b ^ 0xbf, 0x0c ^ 0xcf, 0x0d ^ 0xdf, 0x0e ^ 0xef,
+    0x0f ^ 0xff, 0x10 ^ 0x0a, 0x11 ^ 0x1a, 0x12 ^ 0x2a, 0x13 ^ 0x3a,
+    0x14 ^ 0x4a, 0x15 ^ 0x5a, 0x16 ^ 0x6a, 0x17 ^ 0x7a, 0x18 ^ 0x8a,
+    0x19 ^ 0x9a, 0x1a ^ 0xaa, 0x1b ^ 0xba, 0x1c ^ 0xca, 0x1d ^ 0xda,
+    0x1e ^ 0xea, 0x1f ^ 0xfa};
```

Comment:
Do you think this could be generated directly in the test?

Also, constants must be named `kConst`.

Reply:
> Do you think this could be generated directly in the test?

I didn't want to give the impression that this is something software should do. Instead, software is supposed only ever see the two shares (and not the unmasked key + the mask). But maybe this could also be clarified with a comment.

> Also, constants must be named `kConst`.

I can do that.

Reply:
This is now changed.